### DISCUSSION TITLE
[Feature] merge albums

### DISCUSSION THREAD

**Discussion by akostadinov** [immich-app/immich#13432] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'd like to easily merge albums.

That would be:
- select multiple albums in albums list and have an option to merge them
- select multiple albums when viewing an image that is part of multiple albums and have an option to merge them

Or maybe from the album option list choose "merge" and then be presented with a list of other albums to merge with.

This is very useful after importing a large library and have automatically created many albums.

P.S. opening the albums page is slow when there are many albums.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#3190

Like the template requests, please search for duplicates before opening a new thread.

**Comment by akostadinov** [comment]
Thanks for the link and for working on the project. In my defense, I did search. See a screenshot. Don't assume somebody didn't search just because a RFE is already there :)

![image](https://github.com/user-attachments/assets/b5cccf6d-bd47-4827-ba3a-f941dc8c42dc)

**Reply by bo0tzz** [reply]
It's very common that people don't search and it's the bane of my existence :P Thank you for trying though.

**Comment by theeddie** [comment]
So is there a way to do this ...wanna guess how I found this request
